.make/test.mk

```diff
@@ -274,6 +274,12 @@ coverage-all-html: prebuild-check clean-coverage-overall $(COV_PATH_OVERALL)
 	@go tool cover -html=$(COV_PATH_OVERALL)
 
 # Experimental:
+#
+.PHONY: coveralls
+## Uploads the coverage to coveralls.io
+coveralls: $(GOVERALLS_BIN)
+	cp $(TMP_PATH)/coverage.mode-* profile.cov
+	$(GOVERALLS_BIN) -coverprofile=profile.cov -repotoken=TcANY144GHGlWbepmrjNV4TAn9Hmsm23t -service=jenkins
```

Comment:
No worries @baijum This is just to test if we want such a service or not. I'll post the results on almighty-public.